Automattic/pocket-casts-ios
# Settings Sync: New server values for trim silence podcast setting

| 📘 Part of: #1400 |
|:---:|

Use updated server values for the Trim Silence podcast setting

## To test

* D1: Navigate to a podcast's settings
* D1: Go to Playback Effects
* D1: Enabled "Trim Silence"
* D1: Change the level
* D1: Navigate back to Podcasts list
* D1: Pull to refresh
* D2: Pull to refresh 
* D2: Navigate to a podcast's settings > Playback Effects > Trim Silence
* D2: Verify that settings match

## Checklist
